packages/cli/src/index.ts

```diff
@@ -525,6 +525,7 @@ program
     .option('--vars <vars>')
     .option('--threads <number>')
     .option('--no-version-check')
+    .option('--preview', 'Validate the last preview if available', false)
```

Comment:
for now I'm only doing this on `validate` and not on `deploy` because I?m not sure if that will break some existing deploys for users . we can do it on a separate PR